backend/spec/controllers/spree/admin/orders_controller_spec.rb

```diff
@@ -216,14 +223,15 @@ def initialize(user)
 
       3.times { create(:completed_order_with_totals) }
       expect(Spree::Order.complete.count).to eq 4
-      Spree::Ability.register_ability(OrderSpecificAbility)
 
-      allow(user).to receive_messages has_spree_role?: false
-      spree_get :index
-      expect(response).to render_template :index
-      expect(assigns['orders'].size).to eq 1
-      expect(assigns['orders'].first.number).to eq number
-      expect(Spree::Order.accessible_by(Spree::Ability.new(user), :index).pluck(:number)).to eq  [number]
+      with_ability(OrderSpecificAbility) do
+        allow(user).to receive_messages has_spree_role?: false
+        spree_get :index
+        expect(response).to render_template :index
+        expect(assigns['orders'].size).to eq 1
+        expect(assigns['orders'].first.number).to eq number
+        expect(Spree::Order.accessible_by(Spree::Ability.new(user), :index).pluck(:number)).to eq  [number]
```

Comment:
Line is too long. [107/100]<br>Unnecessary spacing detected.<br>Put one space between the method name and the first argument.